Sources/PackageLoading/PackageBuilder.swift

```diff
@@ -818,8 +818,9 @@ public final class PackageBuilder {
             throw ModuleError.defaultLocalizationNotSet
         }
 
+        // FIXME: use identity instead?
         // The name of the bundle, if one is being generated.
-        let bundleName = resources.isEmpty ? nil : self.manifest.name + "_" + potentialModule.name
+        let bundleName = resources.isEmpty ? nil : self.manifest.displayName + "_" + potentialModule.name
```

Comment:
@neonichu @abertelrud I would have liked to use identity here but not sure about the downstream effects. wdyt?

Reply:
That's a good question — I would say we should probably use the identity, but might need to mangle it (it can contain a solidus, right?)

Reply:
@abertelrud afaik the prior package.manifestName is not mangled specifically - any mangling is done by the usage site

Reply:
Right, this has probably always risked being broken.  But with an identity containing (I think?) a forward slash, we're almost certain to always break it now.  Or am I mistaken about the format of the identity?

Reply:
identity does not contain forward slashes